core/formula/src/main/java/org/csstudio/apputil/formula/math/ACos.java

```diff
@@ -1,9 +1,14 @@
 package org.csstudio.apputil.formula.math;
 
-@SuppressWarnings("nls")
+/**
+ * Acos class see @Math acos function
+ */
 public class ACos extends OneArgMathFunction
 {
-    public ACos()
+	 /**
+     * Constructor
+     */
+	public ACos()
```

Comment:
@abrahamwolk comment from previous PR still applies here and a few other spots

https://github.com/ControlSystemStudio/phoebus/pull/2796#discussion_r1311244806